client/packages/coldchain/src/Equipment/ListView/Toolbar.tsx

```diff
@@ -61,6 +63,36 @@ export const Toolbar = () => {
       name: t('label.non-catalogue'),
       urlParameter: 'isNonCatalogue',
     },
+    {
+      type: 'enum',
```

Comment:
Moved to top and made it default since I thought users would probably like to know which asset is actually functioning or not, etc... but can move if it isn't XD